scripts/helper.js

```diff
@@ -249,6 +258,7 @@ const accountsMetadata = (network) => {
       guardians: contract('guard.seeds', 'guardians'),
       gratitude: contract('gratz.seeds', 'gratitude'),
       pouch: contract('pouch.seeds', 'pouch'),
+      service: contract('servc.seeds', 'service'),
```

Comment:
I suggest this contract to be called "shrine" - i.e. folder of sacred papers

Reply:
@Rieki7  any ideas on the name?